apps/obs-web/src/views/DebrieferWcgopEvaluation.vue

```diff
@@ -302,25 +309,40 @@ export default createComponent({
       });
     }
 
+    onMounted( () => {
+      if (observerMode) {
+        console.log('observer mode')
```

Comment:
yes, but only temporarily.  The app will typically show the logged in user's assessments etc, but since the devs and debriefers that need to see the screens demoed this week aren't observers and don't have assessments, I've hard coded to always load the same observer - should only happen if the logged in user only has the observer role, or if observer mode toggle is active, and will go away in a week or so.